src/python_testing/TC_MCORE_FS_1_3.py

```diff
@@ -57,14 +53,8 @@ def teardown_class(self):
             logging.warning("Stopping app with SIGTERM")
             self.app_process_for_dut_eco.send_signal(signal.SIGTERM.value)
             self.app_process_for_dut_eco.wait()
-        if self.app_process_for_th_eco is not None:
-            logging.warning("Stopping app with SIGTERM")
-            self.app_process_for_th_eco.send_signal(signal.SIGTERM.value)
-            self.app_process_for_th_eco.wait()
 
-        os.remove(self.device_for_dut_eco_kvs)
-        if self.device_for_th_eco_kvs is not None:
-            os.remove(self.device_for_th_eco_kvs)
+        os.remove(self.th_server_kvs)
         super().teardown_class()
 
     async def create_device_and_commission_to_th_fabric(self, kvs, port, node_id_for_th, device_info):
```

Comment:
I the line below this one, [as I have mentioned in the test plan](https://github.com/CHIP-Specifications/chip-test-plans/pull/4575#discussion_r1735006027), we need the command line argument to change from to `th_server_app_path` to `th_server_no_uid_app_path`